coinswap-swift/build-xcframework-ci.sh

```
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+set -euo pipefail
+
+HEADER_BASENAME="CoinswapFFI"
+TARGETDIR="../ffi-commons/target"
+OUTDIR="."
```

Shellcheck (0.11.0)

[warning] OUTDIR appears unused. Verify use (or export if used externally).

(SC2034)